addons/web/static/src/views/view_service.js

```diff
@@ -42,11 +44,19 @@ import { UPDATE_METHODS } from "@web/core/orm_service";
 
 export const viewService = {
     dependencies: ["orm"],
-    start(env, { orm }) {
-        let cache = {};
+    async start(env, { orm }) {
+        let ramCache = {};
```

Comment:
first impression, why is ramcache necessary? i would have guessed that we could use the diskcache instead of having both?

Reply:
then, maybe we could move the ram cache inside the disc cache? so it keeps track of promises itself and user code does not have to do this